### PR TITLE
chore: update demos page with latest demos

### LINKED ISSUE #1444: The suggested blocks plugin is not available on the demo site
**Category**

 - Plugins

**Component**

suggested-blocks

**Describe the bug**

The suggested blocks plugin was never included in the index.md, so it isn't included on the demo site.

https://github.com/google/blockly-samples/blob/master/gh-pages/index.md

**To Reproduce**

1. Go to the demo site.
2. Observe how suggested-blocks does not exist.

**Expected behavior**

The demo should exist.

**Additional context**

It also appears the plugin isn't working locally. Tracking issue: https://github.com/google/blockly-samples/issues/1445

### PR DESCRIPTION
**Pull request by BeksOmega**
### Description

Adds the missing suggested block & interpreter demos to the main page.

Fixes  #1444

### CODE REVIEW

**Review by rachel-fenichel — COMMENTED** [review]
Did you test these plugins with v9.1.1? I did not include them in my testing.

**Comment by BeksOmega** [comment]
Yep I handled all the plugins! I'll double check it + the backwards execution demo though.

**Review by rachel-fenichel — APPROVED** [review]

**Comment by BeksOmega** [comment]
Double checked, and all looks good =)

File: gh-pages/index.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ layout: index
 * [Scroll Options](plugins/scroll-options/test/index.html)
 * [Serialize Disabled Interactions](plugins/serialize-disabled-interactions/test/index.html)
 * [Shadow Block Converter](plugins/shadow-block-converter/test/index.html)
+* [Suggested Blocks](plugins/suggested-block/test/index.html)
```
Comment (vanackej) [inline gh-pages/index.md:34]: There is a typo here, link is incorrect. Should be suggested-blocks with a 's' at the end

Comment (BeksOmega) [reply gh-pages/index.md:34]: Thanks for the catch @vanackej ! Would you be interested in putting up a PR to fix this?

**Pull request merged**